test/data/TR/TRBase.js

```diff
@@ -26,6 +26,14 @@ export const rules = {
                 data: 'wrongCopyrightLink',
                 errors: ['headers.copyright.href-not-match'],
             },
+            {
+                data: 'niceCopyrightException',
```

Comment:
I suggest renaming this section to `copyrightExceptionFound` to be consistent with the next one.
```suggestion
            copyrightExceptionFound: {
```